src/com/palmergames/bukkit/towny/war/siegewar/utils/SiegeWarBlockUtil.java

```diff
@@ -183,4 +183,26 @@ private static boolean isStandingColouredBanner(Block block) {
 				return false;
 		}
 	}
+
+	/**
+	 * This method determines if the supporting block is unstable (e.g. sand,gravel)
+	 *
+	 * @param block the block
+	 * @return true if support block is unstable
+	 */
+	public static boolean isSupportBlockUnstable(Block block) {
```

Comment:
This would actually be pretty useful for FlagWar as well. Thoughts on moving it to a war.common package later?

Reply:
Yes that sounds good :+1: .

In fact there is flag placement stuff which I want to re-use from FlagWar too!
In particular the visual sky marker thing above a flag.

Reply:
Dude, I totally forgot to respond to this when I saw it. Yeah, I'm good with moving the FlagWar beacons to common code.